### ISSUE TITLE
Add uncurated fields during add single inconsistent

### ISSUE THREAD

**Issue by aekhendry** [thesgc/chembiohub_helpdesk#860]
‘Should be possible to add uncurated fields’ : Only possible when drawing a compound, not when entering single inventoryreg item.

**Comment by KarenPorter** [comment]
true.  Either a low priority or amend the testing checklist to reflect the current behaviour.

**Comment by KarenPorter** [comment]
amend testing checklist - <1 hour
